### PR TITLE
[MB-6226] Update service item names content fix

### PR DESCRIPTION
**Pull request by duncan-truss**
## Description

We had consulted with Transcom about the naming of service items a while back and received an [updated content spreadsheet](https://docs.google.com/spreadsheets/d/1c8v94Axdd_0C9Vhg6FbQr5B0a01-rwmcBlyoPZARIm8/edit#gid=0) with what the correct names should be.

This PR creates a migration to update the name column values in the `re_services` table.

Updates were required to the TOO and TIO components that had hardcoded service item names mainly for the basic service items and the auto approved service items depending on the shipment type.

There are going to be a number of Happo diffs caused by updating the storybook mock story data to match what you'd see in the app.  I tried to change most magic strings to use the constants so it would require fewer manual updates going forward.

I didn't update all of the comments since I think they still accurately describe the code we save in the database separate from what we desire the user is shown.  I also tried to change a good number of occurrences of the HHG shipment type to HHG Longhaul because I think that we either set HHG shipments to longhaul or shorthaul.

## Reviewer Notes

The main pages components to review:
1. Move Details page (basic service items)
2. MTO approval modal (basic service items + auto-created accessorial service items)
3. Move Task Order page (accessorial service items)
4. Payment Requests overview for move (basic service items + accessorial service items)
5. Review Payment Request service items (basic service items + accessorial service items)

## Setup

Viewing changes in the office app:
```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

Viewing the storybook changes:
```sh
make storybook
```

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely) have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-6226) for this change

## Screenshots

Screen Shot 2021-02-23 at 1 59 49 PM
Screen Shot 2021-02-23 at 2 00 02 PM
Screen Shot 2021-02-23 at 2 00 12 PM
Screen Shot 2021-02-23 at 2 00 29 PM
Screen Shot 2021-02-23 at 2 01 18 PM
Screen Shot 2021-02-23 at 2 01 26 PM
Screen Shot 2021-02-23 at 2 01 42 PM

### CODE REVIEW

File: src/components/Office/ShipmentServiceItemsTable/ShipmentServiceItemsTable.jsx [hunk 1]
```diff
@@ -4,37 +4,44 @@ import propTypes from 'prop-types';
...
-  domestic_linehaul: 'Domestic linehaul',
-  fuel_surcharge: 'Fuel surcharge',
-  domestic_origin_price: 'Domestic origin price',
-  domestic_destination_price: 'Domestic destination price',
-  domestic_packing: 'Domestic packing',
-  domestic_unpacking: 'Domestic unpacking',
-};
+import { serviceItemCodes } from 'content/serviceItems';
 
 const shipmentTypes = {
```
Comment (duncan-truss) [inline src/components/Office/ShipmentServiceItemsTable/ShipmentServiceItemsTable.jsx:9]: This object determines what we show in the preview before the shipment is approved and send to the prime.  Before we weren't being entirely accurate about which service items were going to be auto-created.  Here is the spreadsheet with the source of truth for shipment types and service items https://docs.google.com/spreadsheets/d/1zeS5J2LVd_cyB2TW2rQPu1hHqKSXT-UY7RpUzfvTi0k/edit#gid=0

File: src/content/serviceItems.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+/* eslint-disable import/prefer-default-export */
+export const serviceItemCodes = {
```
Comment (Ryan-Koch) [inline src/content/serviceItems.js:2]: Would you be opposed to setting the const here and then doing a default export below? That way we can avoid the linter comment. I found a list of examples that I think apply to this rule: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md

File: src/utils/serviceItems.test.js [hunk 3]
```diff
@@ -25,43 +26,43 @@ describe('serviceItems utils', () => {
         const serviceItemCards = [
           {
             id: '1',
-            mtoServiceItemName: 'Counseling services',
+            mtoServiceItemName: serviceItemCodes.CS,
```
Comment (Ryan-Koch) [inline src/utils/serviceItems.test.js:29]: Nice! Thank you for converting these all over to using a constant.

File: src/content/serviceItems.js (hunk 2 above)
Comment (duncan-truss) [reply src/content/serviceItems.js:2]: Yep I'll fix that, wasn't sure if I was going to migrate more content there or not.

**Review by amanda5iu — APPROVED** [review]
Yaaaay, this looks great!

**Review by Ryan-Koch — APPROVED** [review]
This is looking good to me. We chatted about the linter comment thing in comments and it sounds like you've got that in hand, so going to go ahead and hit approve.

**Review by jenniferlynparsons — APPROVED** [review]
Looks good! I love seeing these strings changed over to constants. Makes future changes like this much easier.

**Pull request merged**